bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEPType.java

```diff
@@ -399,6 +400,19 @@
                 }
             }),
 
+    EltakoFRM(RORG._4BS, 0x3f, 0x7f, false, false, "EltakoFRM", 0, A5_3F_7F_EltakoFRM.class, THING_TYPE_ROLLERSHUTTER,
+            0, new Hashtable<String, Configuration>() {
+                private static final long serialVersionUID = 1L;
+                {
+                    put(CHANNEL_ROLLERSHUTTER, new Configuration());
+                    put(CHANNEL_TEACHINCMD, new Configuration() {
+                        {
+                            put(PARAMETER_CHANNEL_TeachInMSG, "fff80d80");
```

Comment:
Are you sure that this rollershutter can be teached in with this message? I have just found in the docs that this actuator sends a MSC message for teach in.

Reply:
I am just ignoring the MSC-telegramm; the teach-in telegram `"fff80d80"` is independent of the data the actuator sent beforehand.
I suppose, correct handling of the MSC-telegramm could enable auto-discovery in openHAB? Currently, when I try auto-discovery and start the teach-in, it just displays the `FRM` as a push-button. I then use the displayed ID, to create the `Thing` manually.